tests/common_utils.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""Utilities used by the tests"""
+
+import pytest
+from pathlib import Path
+from subprocess import run
+from shlex import split
+
+
+def create_config_file(path: Path) -> Path:
+    config_file = f"{path}/default.config"
+    cmd = ["echo", f'repo_location = "{path}"']
+
+    with open(f"{config_file}", "w") as f:
+        run(cmd, stdout=f)
+
+    return config_file
+
+
+def execute_process(cmd: str) -> str:
+    out = run(split(cmd), capture_output=True)
+    return out.stdout.decode("utf-8"), out.stderr.decode("utf-8")
```

Comment:
this instead allows you to use the existing output object and refer to stdout stderr and returncode without defining a different object or returning using tuples
```suggestion
def execute_process(cmd: str) -> str:
    return run(split(cmd), encoding="utf-8", capture_output=True)
```